README.adoc

```diff
@@ -730,6 +730,34 @@ aks_<cluster>_<release>_upgrade_fusion.sh
 
 If you're not running on managed K8s platform such as GKE, AKS, or EKS, you can use Helm to install the Fusion chart to an existing Kubernetes cluster.
 
+For eample, you can run local https://kind.sigs.k8s.io/docs/user/quick-start/[`kind`^] k8s cluster on mac with the following commands.
+
+(To install Fusion locally, you should have a node with at least 12GB of memory, 100GB of disk and 4 CPU cores. The recommended setup is 16GB (or more) of memory and 8+ CPU cores.)
+```
+# assume you have brew installed, if not, please refer to https://brew.sh
+brew install kind
+kind create cluster
+```
+
+Then run the https://github.com/lucidworks/fusion-cloud-native/blob/master/setup_f5_k8s.sh.sh[`setup_f5_k8s.sh`^] script to install Fusion 5.x into local k8s cluster.
+```
+./setup_f5_gke.sh -c <cluster_name> -r <release> -n <namespace> --helm-chart fusion-5.0.3-2.tgz
```

Comment:
I update API version for `logstash` from `apps/v1beta2` to `apps/v1` and generate a tarball locally, but I am not sure what the other easy would be to load this tarball without updating our official chart repo. Heard from @thelabdude, we already have a fix for it, so thinking to check it in here and update to the latest chart when it's released.

Reply:
No, let's not add the chart here ... we need to document the upgrade procedure and then release the updated chart with the logstash fix.

Reply:
removed, just realized there is no such API issue for Kubernetes server that come with Docker Desktop, so updated readme to use that instead of `kind`